### PR TITLE
V2

### PR DESCRIPTION
**Pull request by audn**

### CODE REVIEW

**Review by alii — CHANGES_REQUESTED** [review]
Needs workflow files for ESLint and TypeChecking in CI. Can assist in setting these up if needed.

File: src/common/components/Banner.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+              >
+                <i className={'fas fa-times'} />
+              </div>
+              <div
+                className={
+                  'flex sm:hidden px-3 py-2 w-full bg-brand-primary ring-2 ring-brand-primary' +
+                  ' ring-opacity-80 bg-opacity-50' +
+                  '  mt-6 rounded-md justify-center cursor-pointer hover:bg-brand-primary-dark animate'
+                }
+                onClick={() => onClose()}
```
Comment (alii) [inline src/common/components/Banner.tsx]: This can be `onClick={onClose}`

File: src/common/components/Button.tsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    >
+      {children}
+    </div>
+  );
+};
+const Layout = ({
+  route,
+  children,
+}: {
+  route: string | undefined;
```
Comment (alii) [inline src/common/components/Button.tsx]: `undefined` here is still required. would be better as `route?: string`

Comment (audn) [reply src/common/components/Button.tsx]: Nice catch!

**Pull request merged**